chaostoolkit/cli.py

```diff
@@ -385,6 +404,11 @@ def init(ctx: click.Context, discovery_path: str = "./discovery.json",
 with_plugins(iter_entry_points('chaostoolkit.cli_plugins'))(cli)
 
 
+def is_yaml(experiment_path: str):
```

Comment:
Missing the typing of the return value ;)

```
def is_yaml(experiment_path: str) -> bool:
```

Reply:
done